app/models/miq_expression.rb

```diff
@@ -1429,20 +1429,6 @@ def self.get_entry_details(field)
     end
   end
 
-  def self.is_plural?(field)
-    parts = field.split("-").first.split(".")
-    macro = nil
-    model = model_class(parts.shift)
-    parts.each do |assoc|
-      ref = model.reflection_with_virtual(assoc.to_sym)
-      return false if ref.nil?
```

Comment:
@gtanzillo there's a small change in behavior here - it will blow up if it can't traverse the reflections, whereas the above will return false. Do we want to preserve that?